windowmanagers/2bwm/INSTALL

```diff
@@ -0,0 +1 @@
+make install PREFIX=${INSTALL_ROOT}/usr
```

Comment:
same as above: if the user decides to have `INSTALL_ROOT` as a path containing spaces, that'd break here.

Reply:
Ah sorry, totally forgot about that! I'd quoted it on berry I think, slipped my mind on 2bwm though